bigchaindb/models.py

```diff
@@ -40,20 +40,28 @@ def validate(self, bigchain):
         inputs_defined = all([ffill.tx_input for ffill in self.fulfillments])
```

Comment:
There's a bit of duplication in what I added to this method, but as @r-marques and I discussed, this method is somewhat confused (it's doing too much). Until we get a handle on when and what we should be validating in the server, I think this is OK like this.